Makefile

```diff
@@ -295,11 +294,7 @@ lint-fix: ##@test Run code style checks and fix issues
 
 test: export TARGET := lein
 test: ##@test Run tests once in NodeJS
-	lein with-profile test doo node test once
-
-test-auto: export TARGET := lein
-test-auto: ##@test Run tests in interactive (auto) mode in NodeJS
-	lein with-profile test doo node test
+	yarn shadow-cljs compile mocks && yarn shadow-cljs compile test && node --require ./test-resources/override.js target/test/test.js
```

Comment:
What's the point of these `&&`s? `make` stops on the first command that fails so you can just put teach in their respective lines.

Reply:
Actually, that makes separate Nix shells, so your way is better, though just break lines with trailing `\`.